example/randomCells2D/src/randomCells2D.cpp

```diff
@@ -224,8 +218,9 @@ auto main() -> int
         }
     }
 
+    BufAcc bufAccS{alpaka::zero_memcpy(queue, bufHostS)};
```

Comment:
I have to think more about it, but I am mostly convinced that this approach (with the `zero_copy` returning a buffer) cannot be easily used in the CMS software.

Would it be possible to have an interface like that of `memcpy` ?

Reply:
Ah, I see that it's used [below](https://github.com/alpaka-group/alpaka/pull/1820/files#diff-caf80a938892cfe8b2e7143f6e59b0c82cead0c5583ca324a21a0938a201835bR234)... so why does one need to construct a new buffer here ?

Reply:
Because if I would pass in the destination buffer as argument, I would need to first allocate it, which I try to avoid :)

Reply:
This would indeed be handy in this one case.